File: java 17/llm/src/main/java/com/listenai/llm/models/response/CompletionResponse.java
package com.listenai.llm.models.response;

import com.alibaba.fastjson2.annotation.JSONField;

import java.util.ArrayList;

public class CompletionResponse {
    @JSONField(name = "choices")
    private ArrayList<CompletionChoice> choices;

    @JSONField(name = "created")
    private int createdAt;

    @JSONField(name = "id")
    private String id;

    @JSONField(name = "object")
    private String object;

    @JSONField(name = "usage")
    private SparkUsage usage;

    public ArrayList<CompletionChoice> getChoices() {
        return choices;
    }

    public void setChoices(ArrayList<CompletionChoice> choices) {
        this.choices = choices;
    }

    public int getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(int createdAt) {
        this.createdAt = createdAt;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getObject() {
        return object;
    }

    public void setObject(String object) {
        this.object = object;
    }

    public SparkUsage getUsage() {
        return usage;
    }

    public void setUsage(SparkUsage usage) {
        this.usage = usage;
    }
}

File: .net 6.0/ListenAI.LLM/Models/Response/CompletionResponse.cs
using Newtonsoft.Json;

namespace ListenAI.LLM.Models.Response; 

public class CompletionResponse {
    /// <summary>
    /// AI回复内容
    /// </summary>
    [JsonProperty("choices")]
    public List<CompletionChoice> Choices { get; set; }
    
    /// <summary>
    /// 回复创建时间
    /// </summary>
    [JsonProperty("created")]
    public int CreatedAt { get; set; }
    
    /// <summary>
    /// 回复 ID
    /// </summary>
    [JsonProperty("id")]
    public string Id { get; set; }

    /// <summary>
    /// 回复内容类型 (目前仅限 chat.completion)
    /// </summary>
    [JsonProperty("object")]
    public string Object { get; set; }
    
    /// <summary>
    /// 用量统计
    /// </summary>
    [JsonProperty("usage")]
    public SparkUsage Usage { get; set; }
}